packages/sdk38/src/secp256k1wallet.ts

```diff
@@ -0,0 +1,332 @@
+import {
+  Argon2id,
+  Bip39,
+  EnglishMnemonic,
+  pathToString,
+  Random,
+  Secp256k1,
+  Slip10,
+  Slip10Curve,
+  Slip10RawIndex,
+  stringToPath,
+  xchacha20NonceLength,
+  Xchacha20poly1305Ietf,
+} from "@cosmjs/crypto";
+import { fromBase64, fromHex, fromUtf8, toAscii, toBase64, toHex, toUtf8 } from "@cosmjs/encoding";
+import { assert, isNonNullObject } from "@cosmjs/utils";
+
+import { rawSecp256k1PubkeyToAddress } from "./address";
+import { encodeSecp256k1Signature } from "./signature";
+import { StdSignature } from "./types";
+import {
+  AccountData,
+  Algo,
+  executeKdf,
+  KdfConfiguration,
+  makeCosmoshubPath,
+  OfflineSigner,
+  prehash,
+  PrehashType,
+} from "./wallet";
+
+const serializationTypeV1 = "secp256k1wallet-v1";
+
+/**
+ * A fixed salt is chosen to archive a deterministic password to key derivation.
+ * This reduces the scope of a potential rainbow attack to all CosmJS users.
+ * Must be 16 bytes due to implementation limitations.
+ */
+const cosmjsSalt = toAscii("The CosmJS salt.");
```

Comment:
Why not import this from `wallet.ts`?

Reply:
Sure, thanks. This is a missing deduplication from the `wallet.ts`/`secp256k1wallet.ts` split